engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/ImportsTest.scala

```diff
@@ -77,4 +77,13 @@ class ImportsTest extends PackageTest {
     )) should have message "Compilation aborted due to errors."
     consumeOut should contain("Export statements form a cycle:")
   }
+
+  "Import statements" should "should allow for importing submodules" in {
+    evalTestProject("TestSubmodules") shouldEqual 42
```

Comment:
Is the `TestSubmodules` project evaluated with standard distribution or micro distribution?

Reply:
It doesn't matter TBH. I only import from stdlib println because I want to do additional verification and not deal with unused variable warnings. Strictly speaking it is not necessary because compiler not complaining is already a progress.